src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerDriver.cs

```diff
@@ -727,6 +748,37 @@ internal AnalyzerDriver(ImmutableArray<DiagnosticAnalyzer> analyzers, Func<Synta
             }
         }
 
+        private ImmutableDictionary<DiagnosticAnalyzer, ImmutableArray<CodeBlockAnalyzerAction>> CodeBlockActionsByAnalyzer
+        {
+            get
+            {
+                if (_lazyCodeBlockActionsByAnalyzer == null)
+                {
+                    ImmutableDictionary<DiagnosticAnalyzer, ImmutableArray<CodeBlockAnalyzerAction>> codeBlockActionsByAnalyzer;
+                    var codeBlockActions = this.analyzerActions.CodeBlockActions;
+                    if (codeBlockActions.Any())
```

Comment:
what is type of this.analyzerActions.CodeBLockActions? is it IEnumerable? otherwise, can we use legnth or count explicitly rather than Any?